File: src/main/java/org/minerva/stateservice/hrm/repos/OrgRepos.java
package org.minerva.stateservice.hrm.repos;

import org.minerva.stateservice.hrm.models.Org;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface OrgRepos extends JpaRepository<Org, Long> {
}

File: src/main/java/org/minerva/stateservice/hrm/repos/OrgRepos.kt
package org.minerva.stateservice.hrm.repos

import org.minerva.stateservice.hrm.models.Org
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface OrgRepos : JpaRepository<Org?, Long?> {
    fun findByAncestry(ancestry: String?): List<Org>
}